desktop/src/main/java/bisq/desktop/primary/main/content/social/SocialView.java

```diff
@@ -17,30 +17,50 @@
 
 package bisq.desktop.primary.main.content.social;
 
-import bisq.desktop.common.view.NavigationTarget;
-import bisq.desktop.common.view.TabView;
+import bisq.desktop.common.view.*;
+import bisq.desktop.overlay.OverlayWindow;
+import bisq.desktop.primary.main.content.social.gettingStarted.GettingStartedView;
 import bisq.i18n.Res;
+import javafx.beans.value.ChangeListener;
+import javafx.scene.Parent;
+import javafx.scene.layout.Region;
 import lombok.extern.slf4j.Slf4j;
 
 @Slf4j
 public class SocialView extends TabView<SocialModel, SocialController> {
-
+    protected final ChangeListener<View<? extends Parent, ? extends Model, ? extends Controller>> viewChangeListener;
+    
     public SocialView(SocialModel model, SocialController controller) {
         super(model, controller);
 
-        headlineLabel.setText(Res.get("welcome"));
-
         addTab(Res.get("social.gettingStarted"), NavigationTarget.GETTING_STARTED);
         addTab(Res.get("social.discuss"), NavigationTarget.DISCUSS);
         addTab(Res.get("social.learn"), NavigationTarget.LEARN);
         addTab(Res.get("social.connect"), NavigationTarget.CONNECT);
+
+        setupHeadline(model.getView()); // TODO: check why view is null at the beginning
+        viewChangeListener = (observable, oldValue, newValue) -> setupHeadline(newValue);
     }
 
     @Override
     protected void onViewAttached() {
+        model.getView().addListener(viewChangeListener);
     }
 
     @Override
     protected void onViewDetached() {
+        model.getView().removeListener(viewChangeListener);
+    }
+    
+    private void setupHeadline(Object view) {
```

Comment:
> I don't know the current design, but shouldn't the green line under the selected tab be closer to the text?

"Welcome" headline on [newest design](https://xd.adobe.com/view/88ec6f20-15ca-4326-81f4-30a30cc1aa0a-ddbf/screen/2003f6b1-f8dd-4b46-87ac-c80b6c92bb2b) for this specific screen is bigger. It currently causes UI issues like this mentioned by @ripcurlx. Fixing them requires reorganization of [TabView](https://github.com/bisq-network/bisq2/blob/1f3507ec170121e736a2c0f53db8997553f7895e/desktop/src/main/java/bisq/desktop/common/view/TabView.java#L57) - I'm aware of it however I didn't wanted to make it in this pull request - I predict that designs may still evolve. If you thinks I should do it here just let me know.

CC: @chimp1984

Reply:
Can you ask Pedro if the large distance of the line is intended? It looks to me to much distance, but he is our designer, so if he wants it that way we do it. Yes we can do that in an upcoming PR.

Reply:
I'll fix distance between line and "Welcome" label in an upcoming PR - it's a bit challenging issue.